src/views/Alteration.vue

```diff
@@ -193,6 +206,35 @@ export default class Alteration extends Mixins(CommonMixin, LegalApiMixin, Filin
     return this.getAppValidate && !this.getAlterationValidFlags.isValidFileNum
   }
 
+  private get filingFeesPrice (): string {
+    if (this.getFeePrices?.filingFees) {
+      return `$${this.getFeePrices.filingFees.toFixed(2)}`
+    }
+    return ''
+  }
+
+  private get futureEffectiveFeesPrice (): string {
+    if (this.getFeePrices?.futureEffectiveFees) {
+      return `$${this.getFeePrices.futureEffectiveFees.toFixed(2)}`
+    }
+    return ''
+  }
+
+  private emptyFees: FeesIF = {
```

Comment:
Move this to where the Fees IF is declared. Then anyone using the IF also has the empty IF it can use :)

Also it probably makes more sense for all of these to be '' or 0 (depending on their type).

Reply:
ok, moved. About the null values, I think it's better to keep it as null for the `emptyFees` because I know that it's not coming from pay-api (the prices may be zero in some future scenario).

Reply:
OK